src/test/java/WebDriverSeleniumTests/GoogleSearchForWikiEdge.java

```diff
@@ -0,0 +1,91 @@
+package WebDriverSeleniumTests;
+
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.*;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import ru.yandex.qatools.ashot.AShot;
+import ru.yandex.qatools.ashot.Screenshot;
+import ru.yandex.qatools.ashot.shooting.ShootingStrategies;
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class GoogleSearchForWikiEdge {
+
+    @Test
+    public void GoogleSearchForWikiIsNotEmpty() throws IOException {
+
+        final Logger logger = Logger.getGlobal();
+
+
+        System.setProperty("webdriver.edge.driver", "D:\\Tools\\msedgedriver.exe");
```

Comment:
All three drivers should be placed into project. I don't have D:\\Tools\\msedgedriver.exe on my laptop, so if I will start your tests - they will fail

Reply:
Placed all 3 drivers into resources folder